composer.json

```diff
@@ -7,15 +7,17 @@
             "EventSauce\\MessageOutbox\\": "src/MessageOutbox",
             "EventSauce\\MessageOutbox\\IlluminateOutbox\\": "src/IlluminateOutbox",
             "EventSauce\\MessageOutbox\\DoctrineOutbox\\": "src/DoctrineOutbox",
-            "EventSauce\\MessageRepository\\IlluminateMessageRepository\\": "src/IlluminateMessageRepository",
-            "EventSauce\\MessageRepository\\DoctrineMessageRepository\\": "src/DoctrineMessageRepository",
-            "EventSauce\\MessageRepository\\TestTooling\\": "src/MessageRepositoryTestTooling"
+            "EventSauce\\IlluminateMessageRepository\\": "src/IlluminateMessageRepository",
```

Comment:
These namespaces should not change.

Reply:
sorry, not on purpose. I will revert that